privacyidea/static/templates/menu.html

```diff
@@ -158,3 +158,11 @@
 <div class="alert alert-warning" ng-show="logoutWarning" translate>
     You will be logged out in {{ myCountdown }} second(s).
 </div>
+<div class="alert alert-warning {{ class_subscription_expired }}"
```

Comment:
What is this css_class good for? Are you randomizing it, so that it can not be hidden?

Reply:
that was the intention. It can be hidden, but pops up again. If you do it correctly, you can also hide it permanently.